### PR TITLE
Fix issue link

### PR DESCRIPTION
**Pull request by johnathanludwig**
The link was going to the wrong PR.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Breaking changes:
 Features:
 
 - [#1291](https://github.com/rails-api/active_model_serializers/pull/1291) Add logging (@maurogeorge)
-- [#1225](https://github.com/rails-api/active_model_serializers/pull/1125) Better serializer lookup, use nested serializer when it exists (@beauby)
+- [#1225](https://github.com/rails-api/active_model_serializers/pull/1225) Better serializer lookup, use nested serializer when it exists (@beauby)
```
Comment (bf4) [inline CHANGELOG.md]: Awesome, thanks! While you're in there, would you mind fixing

`[#tbd](https://github.com/rails-api/active_model_serializers/pull/tbd) Rename ArraySerializer to` ...

to refer to https://github.com/rails-api/active_model_serializers/pull/1251 ?

Comment (johnathanludwig) [inline CHANGELOG.md]: Done! :)

**Comment by bf4** [comment]
Would you mind squashing the two commits and force pushing?  If not, it's ok

**Comment by johnathanludwig** [comment]
... well shit. I've screwed that up. I was just editing from the github web UI before. I haven't squashed and force pushed before.

I'll make a new brach/pr with one commit.

**Comment by johnathanludwig** [comment]
Replaced by #1364

**Comment by bf4** [comment]
@johnathanludwig No worries.  In the future, know that I can do it for you, and there's rarely a need to open a new PR (except via the GitHub web ui since you can't do anything fancy like amend there)

**Pull request closed without merging**